### ISSUE TITLE
PHP 7.0.8 libapache2-mod-php7.0 postinst broken

### ISSUE THREAD

**Issue by xpender** [gplessis/dotdeb-php#139]
I looks like that in the package libapache2-mod-php7.0 Version 7.0.8-1~dotdeb+8.1 the postinst script is broken.

```
Preparing to unpack .../libapache2-mod-php7.0_7.0.8-1~dotdeb+8.1_amd64.deb ...
Unpacking libapache2-mod-php7.0 (7.0.8-1~dotdeb+8.1) ...
Setting up libapache2-mod-php7.0 (7.0.8-1~dotdeb+8.1) ...

Creating config file /etc/php/7.0/apache2/php.ini with new version
/var/lib/dpkg/info/libapache2-mod-php7.0.postinst: 291: [: !=: unexpected operator
apache2_switch_mpm prefork: No action required
apache2_invoke: Enable module php7.0
apache2_reload: Your configuration is broken. Not restarting Apache 2
apache2_reload: [Thu Jul 14 15:13:14.470138 2016] [:crit] [pid 25755:tid 140478884919168] Apache is running a threaded MPM, but your PHP Module is not compiled to be threadsafe.  You need to recompile PHP.
apache2_reload: AH00013: Pre-configuration failed
```

mpm_event stays enabled, so this results in an broken Apache2 Configuration.

Manual fix that works for now:
```
a2dismod mpm_event
a2enmod mpm_prefork
```

**Comment by gplessis** [comment]
I'm still working on providing the ZTS version of PHP 7.0. In the meantime, mod_php7.0 should only be used with the `prefork` MPM. I won't enforce the dependency though.

**Comment by xpender** [comment]
It's not that I want to use the ZTS version or `event` MPM.

Before the postinst script automatically switched to `prefork` MPM if it was not active, but there is now a shell script error which breaks this:

```/var/lib/dpkg/info/libapache2-mod-php7.0.postinst: 291: [: !=: unexpected operator```

If I remember correctly, with older 7.0.X versions the auto-switch worked fine.

Further informations here on running OS:

```Linux ip-10-32-89-71 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt25-2 (2016-04-08) x86_64 GNU/Linux```

```Distributor ID:	Debian
Description:	Debian GNU/Linux 8.5 (jessie)
Release:	8.5
Codename:	jessie```

```
ii  php7.0-bcmath     7.0.8-1~dotdeb+8.1           amd64        Bcmath module for PHP
ii  php7.0-cli        7.0.8-1~dotdeb+8.1           amd64        command-line interpreter for the PHP scripting language
ii  php7.0-common     7.0.8-1~dotdeb+8.1           amd64        documentation, examples and common module for PHP
ii  php7.0-curl       7.0.8-1~dotdeb+8.1           amd64        CURL module for PHP
ii  php7.0-gd         7.0.8-1~dotdeb+8.1           amd64        GD module for PHP
ii  php7.0-igbinary   1.2.1-1~2b7c703-1~dotdeb+8.2 amd64        igbinary serializer for PHP
ii  php7.0-intl       7.0.8-1~dotdeb+8.1           amd64        Internationalisation module for PHP
ii  php7.0-json       7.0.8-1~dotdeb+8.1           amd64        JSON module for PHP
ii  php7.0-mbstring   7.0.8-1~dotdeb+8.1           amd64        MBSTRING module for PHP
ii  php7.0-mcrypt     7.0.8-1~dotdeb+8.1           amd64        libmcrypt module for PHP
ii  php7.0-memcached  2.2.0-2-52c644b-1~dotdeb+8.2 amd64        memcached extension module for PHP, uses libmemcached
ii  php7.0-mongodb    1.1.8-1~dotdeb+8.1           amd64        MongoDB driver for PHP
ii  php7.0-msgpack    2.0.1-1~dotdeb+8.2           amd64        MessagePack serializer for PHP
ii  php7.0-mysql      7.0.8-1~dotdeb+8.1           amd64        MySQL module for PHP
ii  php7.0-opcache    7.0.8-1~dotdeb+8.1           amd64        Zend OpCache module for PHP
ii  php7.0-readline   7.0.8-1~dotdeb+8.1           amd64        readline module for PHP
ii  php7.0-redis      3.0.0~rc1-1~dotdeb+8.2       amd64        PHP extension for interfacing with Redis
ii  php7.0-sqlite3    7.0.8-1~dotdeb+8.1           amd64        SQLite3 module for PHP
ii  php7.0-xml        7.0.8-1~dotdeb+8.1           amd64        DOM, SimpleXML, WDDX, XML, and XSL module for PHP
ii  php7.0-xmlrpc     7.0.8-1~dotdeb+8.1           amd64        XMLRPC-EPI module for PHP
```

**Comment by gplessis** [comment]
Got it! Thanks for the report, I'll work on it asap.

**Comment by gplessis** [comment]
A fix has been pushed to the 7.0.9 version. Could you please confirm?

**Comment by xpender** [comment]
I can confirm this is fixed..

I tested it on a freshly installed jessie and a couple of already running machines, postinst works fine and is auto-switching to prefork mpm if not already enabled.
